operator/internal/central/extensions/reconcile_tls.go

```diff
@@ -63,6 +66,9 @@ type createCentralTLSExtensionRun struct {
 	caRotationAction carotation.Action
 	centralObj       *platform.Central
 	currentTime      time.Time
+
+	issueCentralCertFn func(map[string][]byte, mtls.CA, ...mtls.IssueCertOption) error
+	issueServiceCertFn func(map[string][]byte, mtls.CA, mtls.Subject, string, ...mtls.IssueCertOption) error
```

Comment:
If injecting the `WithValidityNotBefore` is the only use case, then how about instead providing a (single) `extraIssueCertOptions` or `issueCertOptionsForTesting` field?
Perhaps that would be cleaner than overriding two functions. Also this way we would not need to touch the `&createCentralTLSExtensionRun{...}` in the production code.

Reply:
Done